### PR TITLE
#2 documentation

### PR DESCRIPTION
**Pull request by lil5**
Added documentation

Please follow the link from [here](https://github.com/tretapey/raisincss/issues/2#issuecomment-578614196)

### CODE REVIEW

File: docs/how-to-use.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+If you want to customize for your own use feel free to fork or download this repo, the `.scss` files has everything you need.
+
+```scss
+@import '~raisincss/raisin';
+```
+
+If you need to change any of the variables please add changed variables above the import.
+
+Or import each file apon requirements.
```
Comment (tretapey) [inline docs/how-to-use.md]: typo here, it should say "upon" instead of "apon"

File: README.md [hunk 2]
```diff
@@ -8,730 +8,7 @@ An Utility CSS only library.
 - Skeleton and utility based tool
 - Lightweight and easy to use
```
Comment (tretapey) [inline README.md:10]: Maybe add the link the the documentation here

File: docs/index.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# RaisinCSS
+
+An Utility CSS only library.
+
+- Supports CSS Grid (Check grid section)
+- Supports Flexbox
+- Totally open to customization
+- Skeleton and utility based tool
+- Lightweight and easy to use
+
```
Comment (tretapey) [inline docs/index.md:10]: @lil5  here we should add the table of contents with links to each doc file.

**Comment by lil5** [comment]
Is there anything I’m missing @tretapey?

**Review by tretapey — COMMENTED** [review]
> Is there anything I’m missing @tretapey?

Hi @lil5, yes, please check my comments in the PR.

**Comment by tretapey** [comment]
also please do a pull from master to check for conflicts

**Comment by tretapey** [comment]
Closing this because no activity in the past months

**Pull request closed without merging**